docs/group_getting_started/ggs_signup.rst

```diff
@@ -16,38 +16,35 @@ Step 1: Sign up for Solace PubSub+ Cloud
 To sign up for Solace PubSub+ Cloud, `click here <https://cloud.solace.com/signup/>`_ to create a **free** account. Click on the following video to view an example of
 signing up.
 
-.. raw:: html
-
-	<video width="768" height="480" muted poster="../_static/img/signup.jpg" onclick="this.paused ? this.play() : this.pause();" style="cursor: pointer;">
-		<source src="../_static/video/signup.webm">
-	</video>
+.. image:: ../img/signup.jpeg
+    :height: 550px
 
 Once you have submitted your details, you will receive a confirmation email. By clicking the Activate button in the email, you will be brought to the Solace PubsSub+ Cloud Console
 where you must enter a password to activate your Solace PubSub+ Cloud account. Once activated, you will be automatically logged into the console.
 
+.. image:: ../img/activation-email.jpeg
+    :height: 500px
+
+.. _`step 2`:
+
 Step 2: Create a service
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Once you have logged into the console, you will be brought to the My Services page. This page will show you all of the various services you have created in Solace PubSub+ Cloud. To
-create your first service, press the Create Service button.
-
-This brings you to the Create Service page. Here you must select a plan type, enter a service name, and choose the cloud provider and region where you would like to create your
-service.
+create your first service, follow the steps below.
 
-Solace PubSub+ Cloud gives you access to different plan types that will give various levels of messaging performance. By default, a new Solace PubSub+ Cloud account only has access
-to the free plan. Later on, you may request access to additional plans by selecting them and pressing the request access button. For our first service, let's select the Free Plan.
-
-Services are identified in the console by their name. Enter a service name of **My First Service**.
-
-Your service will be created in the cloud provider and region of your choice. By pressing on the map, you can choose one of Solace PubSub+ Cloud's available regions. If the 
-region you prefer isn't available, you can contact Solace PubSub+ Cloud support and ask for your region to be supported. For your first service, choose the region closest to 
-where your application will run. You can then complete the process by pressing the Start Service button.
+1. Press the Create Service button.
+2. You must then select a **plan type**, enter a **service name**, and choose the **cloud provider and region** where you would like to create your service.
+3. For your first service, select the **Free Plan**. Solace PubSub+ Cloud gives you access to different plan types that will give various levels of messaging performance. By default, a new Solace PubSub+ Cloud account only has access to the free plan. Later on, you may request access to additional plans by selecting them and pressing the request access button.
```

Comment:
Could we say Free or Nano depending on your account setup. Some people will not see the "Free Plan".